source/common/stream_info/utility.h

```diff
@@ -81,6 +116,42 @@ class ResponseFlagUtils {
       FlagStringAndEnum{OVERLOAD_MANAGER, ResponseFlag::OverloadManager},
   };
 
+  static constexpr std::array ALL_RESPONSE_FULL_STRING_FLAGS{
```

Comment:
Instead of a separate array, can you make this a single array (combined with above) with type std::pair (or struct with better field names) for the short and long name? That will prevent errors when adding a flag if someone only adds to one array and not the other.

Reply:
Good callout, thanks